tests/org.eclipse.n4js.jsdoc.tests/.classpath

```diff
@@ -3,5 +3,6 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="xtend-gen"/>
```

Comment:
the project folder needs the new "xtend-gen" folder to be checked in (otherwise there is an error after checkout).
also, the build.properties needs an additional entry to avoid a warning.